src/musort.py

```diff
@@ -80,28 +82,19 @@ def get_files_recursive(self, directory):
     def get_compatible(self):
         music = []
         for file in self.files:
-            match file.split("."):
-                case [*_, "flac"]:
-                    music.append(file)
-                case [*_, "mp3"]:
-                    music.append(file)
-                case [*_, "mp1"]:
-                    music.append(file)
-                case [*_, "mp2"]:
-                    music.append(file)
-                case [*_, "opus"]:
-                    music.append(file)
-                case [*_, "ogg"]:
-                    music.append(file)
-                case [*_, "wma"]:
-                    music.append(file)
+            file_extension = file.split(".")[-1]
+
+            if file_extension in supported_formats:
```

Comment:
in statement checks if supported_formats(array) has file_extension

Reply:
Nice optimizations! Merging your pr!